test/test_response.py

```diff
@@ -35,6 +35,8 @@
 from urllib3.util.response import is_fp_closed
 from urllib3.util.retry import RequestHistory, Retry
 
+OptInt: typing.TypeAlias = int | None
```

Comment:
Unlike other typing stuff not evaluated because of `from __future__ import annotations`, `int | None` here is so tests in Python 3.8 and 3.9 fail